src/ControlGallery/src/Xamarin.Forms.ControlGallery.WindowsUniversal/MainPage.xaml.cs

```diff
@@ -30,38 +29,66 @@ public MainPage()
 		{
 			InitializeComponent();
 
-			if (Xamarin.Forms.ControlGallery.WindowsUniversal.App.RunningAsUITests)
-			{
-				var bounds = ApplicationView.GetForCurrentView().VisibleBounds;
-				var scaleFactor = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
-				var size = new Windows.Foundation.Size(bounds.Width * scaleFactor, bounds.Height * scaleFactor);
-				ApplicationView.PreferredLaunchViewSize = size;
-				ApplicationView.PreferredLaunchWindowingMode = ApplicationViewWindowingMode.PreferredLaunchViewSize;
-			}
+			// some tests need to window to be large enough to click on things
+			// can we make this only open to window size for UI Tests?
+			//var bounds = ApplicationView.GetForCurrentView().VisibleBounds;
+			//var scaleFactor = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
+			//var size = new Windows.Foundation.Size(bounds.Width * scaleFactor, bounds.Height * scaleFactor);
+			//ApplicationView.PreferredLaunchViewSize = size;
+			//ApplicationView.PreferredLaunchWindowingMode = ApplicationViewWindowingMode.PreferredLaunchViewSize;
 
 
-			_app = new Controls.App();
 
 			// When the native control gallery loads up, it'll let us know so we can add the nested native controls
 			MessagingCenter.Subscribe<NestedNativeControlGalleryPage>(this, NestedNativeControlGalleryPage.ReadyForNativeControlsMessage, AddNativeControls);
 
 			// When the native binding gallery loads up, it'll let us know so we can set up the native bindings
 			MessagingCenter.Subscribe<NativeBindingGalleryPage>(this, NativeBindingGalleryPage.ReadyForNativeBindingsMessage, AddNativeBindings);
 
+		}
+
+		public void LoadApplication()
```

Comment:
Because of how the dispatchers get initialized this is called during LoadApplication from the App after the dispatchers are setup